src/SFA.DAS.AssessorService.Web.Staff/Startup.cs

```diff
@@ -63,6 +64,9 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
             });
 
             services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
+            services.AddMvc()
+                .AddMvcOptions(m => m.ModelMetadataDetailsProviders.Add(new HumanizerMetadataProvider()))
+                .SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
```

Comment:
Looks like .AddMvc() is being called twice now.  Not sure if that makes any difference, but still.  🤷‍♂️

Reply:
Hoisted by my own petard!